app/controllers/MovementController.scala

```diff
@@ -54,7 +55,12 @@ class MovementController @Inject()(
         Future.successful(BadRequest(choice_page(appConfig, formWithErrors))),
       form =>
         customsCacheService.cache[ChoiceForm](appConfig.appName, choiceId, form).map { _ =>
-          Redirect(controllers.routes.MovementController.displayDucrPage())
+            if(form.choice.toString == "SMP") {
```

Comment:
Instead of `if` my propose is to use pattern matching. In the future, we will have three different journeys (arrival and departure have the same journey, always redirection to ducr page)